msg/vehicle_odometry.msg

```diff
@@ -0,0 +1,34 @@
+# Vehicle odometry data. Fits ROS REP 147 for aerial vehicles
+uint64 timestamp		# time since system start (microseconds)
+
+# Position in NED earth-fixed frame (meters). NaN if invalid/unknown
```

Comment:
The description in this message indicates that this is always NED. Is this always the case? If not, then the EKF needs to know and we need a means of communicating that information.

Reply:
@priseborough this is always the case for the PX4 side for now.